.circleci/config.yml

```diff
@@ -257,8 +255,8 @@ jobs:
       - run:
          command: |
             sudo rm -rf /usr/local/go
-            wget https://dl.google.com/go/go1.18.1.linux-arm64.tar.gz
-            sudo tar -C /usr/local -xzvf go1.18.1.linux-arm64.tar.gz
+            wget https://golang.org/dl/go${GO_VERSION}.linux-arm64.tar.gz
```

Comment:
Small, but this was updated to use `dl.google.com` back in #12818 since it is the canonical location for the tarball and there are two redirects between it and `golang.org/dl`.

Reply:
Thank you, I did a manual merge and did not see the update. I will change it using the GO_VERSION env var to ensure we don't have to change a lot of things when we upgrade go version.